File: DSA/SakshamKashyap_590014183/day16/day16_Question_1.py
#prompt the user to enter the array

My_Array = list(map(int,input("Enter the elements (give space between every element): ").strip().split()))

target_value = int(input("Enter the number you want to find in the array: "))

found = False

for i in range(len(My_Array)):
    if My_Array[i] == target_value:
        print(f"the traget element index is: {i}")
        found = True
        break
if not found:
    print("-1") # Target value not found in the array

File: DSA/Rahul_Mankani_590014725/Day16_Question1.c
#include <stdio.h>

int main() 
{
    int arr[100];
    int n;
    int target;
    int index = -1;
    printf("Enter number of elements: ");
    scanf("%d", &n);
    if (n <= 0 || n > 100) 
    {
        printf("Invalid size! Enter between 1 and 100.\n");
        return 1;
    }
    printf("Enter elements: ");
    for (int i = 0; i < n; i++) 
    {
        scanf("%d", &arr[i]);
    }
    printf("Enter number to search: ");
    scanf("%d", &target);
    for (int i = 0; i < n; i++) 
    {
        if (arr[i] == target) 
        {
            index = i;
            break;
        }
    }
    if (index != -1) 
    {
        printf("Number %d found at index %d.\n", target, index);
    } 
    else 
    {
        printf("Number %d not found in array.\n", target);
    }
    return 0;
}
